benchmarks/NonlinearProblem/nonlinear_solver_23_tests.jmd

```diff
@@ -100,10 +100,10 @@ function benchmark_problem!(prob_name; solver_tracker=solver_tracker, selected_N
     solvers_TR = filter(s -> s.type==:TR, successful_solvers)
     solvers_LM = filter(s -> s.type==:LM, successful_solvers)
     solvers_PT = filter(s -> s.type==:PT, successful_solvers)
```

Comment:
@yonatanwesen why was this added as a separate solver list? It is a single solver so we don't need a separate plot. Can you open a follow up PR to fix this